src/pages/components/menu-buttons/usage.mdx

```diff
@@ -0,0 +1,621 @@
+this button does have particular use where screen real estate is limited and one
+of the actions has particular importance.
+
+Combo button has only one button type: primary.
+
+<Row>
+<Column colLg={8}>
+
+![An example of combo button](images/usage-menu-combo-button-1.png)
+
+</Column>
+</Row>
+
+### Best practices
+
+#### Chose a default action
+
+Remember to choose the default, primary action that will be displayed in the
+primary button so it is not hidden within the menu of additional actions. The
+primary default action is typically the most commonly used action there is to
+take.
+
+<Row>
+<Column colLg={8}>
+
+![Choose a default action](images/usage-menu-combo-button-2.png)
+
+<Caption>
+  Example shows that users mostly choose to save records, while less common
+  options are hidden as extra actions.
+</Caption>
+</Column>
+</Row>
+
+#### Reduce visual complexity on a page
+
+Combo buttons reduce visual complexity by grouping similar commands together.
+For example, how navigation menus group together related options to enable
+conceptual understanding of the site information structure.
+
+<Row>
+<Column colLg={8}>
+
+![Reduce visual complexity on a page](images/usage-menu-combo-button-3.png)
+
+<Caption>
+  Example shows that many actions (ghost buttons) applying to the same data can
+  be replaced by the combo button.
+</Caption>
+</Column>
+</Row>
+
+## Overflow menu
+
+Overflow menu is also another option that is available to the user when there is
+a space constraint, usually used in data table rows or in tiles. Overflow menu
+has the ghost button style only.
+
+<Row>
+<Column colLg={8}>
+
+![An example of overflow menu](images/usage-overflow-1.png)
+
+</Column>
+</Row>
+
+### Best practices
+
+#### Overflow menu in cards
+
+When there are more than three interactive icons, consider moving less important
+ones into the overflow menu. The overflow menu then takes the place of a third
+action.
+
+<Row>
+<Column colLg={8}>
+
+![An example of overflow menu in cards](images/usage-overflow-2.png)
+
+</Column>
+</Row>
+
+#### Overflow menu icon
+
+Overflow icon can be customized to align with the product's requirements.
+However, it's advisable to make minimal changes to the overflow button's icons
+to maintain consistency across the product and ensure a consistent user
+experience.
+
+<Row>
+<Column colLg={4}>
+
+![An example of another overflow menu icon](images/usage-overflow-3.png)
+
+</Column>
+</Row>
+
+#### Overflow versus tab tip (popover)
+
+Overflow menu displays extra options with simple text, typically utilized within
+data table rows, tiles, or cards.
+[Tab tip](https://carbondesignsystem.com/components/popover/usage/#tab-tip)
+popover provides more complex inputs such as a checkbox group or a radio button
+group, usually connected to a data table toolbar.
+
+<Row>
+<Column colLg={4}>
+
+![An example of overflow menu](images/usage-overflow-menu.png)
```

Comment:
The bottom border line of the left image does not extend all the way to the right edge of the illustration.